main.go

```diff
@@ -254,13 +254,13 @@ func GetFileTimes(path string) (atime, mtime, ctime time.Time, err error) {
 	stat := fileInfo.Sys().(*syscall.Stat_t)
 
 	// Extract access time (atime)
-	atime = time.Unix(stat.Atim.Sec, stat.Atim.Nsec)
+	atime = time.Unix(stat.Atimespec.Sec, stat.Atimespec.Nsec)
```

Comment:
One thing about these changes... and this code in  general, is that it means locar can *only* work on UNIX-like systems.  I guess that's ok for now.